src/index/Index.h

```diff
@@ -414,14 +428,14 @@ class Index {
 
   static pair<FullRelationMetaData, BlockBasedRelationMetaData> writeRel(
       ad_utility::File& out, off_t currentOffset, Id relId,
-      const vector<array<Id, 2>>& data, bool functional);
+      const MmapVector<array<Id, 2>>& data, size_t distinctC1, bool functional);
```

Comment:
Can you add a method comment for this and I'm guessing `distinctC1` is the number of distinct elements in the first column, right?